components/sst_importer/src/sst_importer.rs

```diff
@@ -292,12 +334,249 @@ impl SstImporter {
         Ok(())
     }
 
+    pub fn shrink_by_tick(&self) -> usize {
+        let mut shrink_buff_size: usize = 0;
+        let mut retain_buff_size: usize = 0;
+        let mut shrink_files: Vec<PathBuf> = Vec::default();
+        let mut retain_file_count = 0_usize;
+
+        self.file_locks.retain(|_, (c, start)| {
+            let mut need_retain = true;
+            match c {
+                CacheKvFile::Mem(buff) => {
+                    let buflen = buff.len();
+                    // The term of recycle memeory is 60s.
+                    if c.ref_count() == 1 && c.is_expired(start) {
+                        need_retain = false;
+                        shrink_buff_size += buflen;
+                    } else {
+                        retain_buff_size += buflen;
+                    }
+                }
+                CacheKvFile::Fs(path) => {
+                    let p = path.to_path_buf();
+                    // The term of recycle file is 10min.
+                    if c.ref_count() == 1 && c.is_expired(start) {
+                        need_retain = false;
+                        shrink_files.push(p);
+                    } else {
+                        retain_file_count += 1;
+                    }
+                }
+            }
+
+            need_retain
+        });
+
+        if self.import_support_download() {
+            let shrink_file_count = shrink_files.len();
+            info!("shrink space by tick"; "shrink files count" => shrink_file_count, "retain files count" => retain_file_count);
```

Comment:
Why need to be `info`? I can see log file only has this logs in an idle cluster.

> [sst_importer.rs:442] ["shrink cache by tick"] ["retain size"=0] ["shrink size"=0]

The log seems useless and meaningless.